greenboot.spec

```diff
@@ -37,8 +37,10 @@ Obsoletes:          greenboot-rpm-ostree-grub2 <= 0.12.0
 %{summary}.
 
 %package default-health-checks
-Summary:            Update platforms DNS resolution and connection check for greenboot
+Summary:            Series of optional and curated health checks
```

Comment:
@LorbusChris @nullr0ute I'm not 100% content with this, any feedback/ideas?

Reply:
drop the "optional and" so just have it as something like "Default curated health checks"

Reply:
Done